docs/governance/index.md

```diff
@@ -1,31 +1,127 @@
+- [Maatregelen van gebruiksverantwoordelijken voor gebruik](https://minbzk.github.io/Algoritmekader/vereisten/maatregelen_van_gebruiksverantwoordelijken_voor_gebruik/index.html)
+- [Melden van ernstige incidenten](https://minbzk.github.io/Algoritmekader/vereisten/melding_ernstige_incidenten/index.html)
+- [Natuurlijke personen die menselijk toezicht uitvoeren zijn bekwaam, opgeleid, beschikken over autoriteit en krijgen ondersteuning](https://minbzk.github.io/Algoritmekader/vereisten/gebruiksverantwoordelijken_menselijk_toezicht_natuurlijke_personen/index.html)
+- [Proportionaliteit en subsidiariteit](https://minbzk.github.io/Algoritmekader/vereisten/beginsel_van_proportionaliteit_en_subsidiariteit/index.html)
+- [Recht op uitleg AI-besluiten](https://minbzk.github.io/Algoritmekader/vereisten/Recht_of_uitleg_AI-besluiten/index.html)
+- [Registratieverplichtingen voor aanbieders van AI-systemen met een hoog risico](https://minbzk.github.io/Algoritmekader/vereisten/registratieverplichtingen_hoog_risico/index.html)
+- [Relevante feiten en belangen zijn bekend](https://minbzk.github.io/Algoritmekader/vereisten/zorgvuldigheidsbeginsel/index.html)
+- [Technische documentatie voor hoog-risico AI](https://minbzk.github.io/Algoritmekader/vereisten/technische_documentatie_voor_hoog_risico_ai/index.html)
+- [Veilig melden van inbreuk op AI verordening](https://minbzk.github.io/Algoritmekader/vereisten/recht_op_uitleg_ai_besluiten/index.html)
+- [Verantwoordelijkheden worden toegewezen en beschreven](https://minbzk.github.io/Algoritmekader/vereisten/beschrijven_en_toewijzen_van_verantwoordelijkheden_bij_verwerking_persoonsgegevens/index.html)
+- [Verantwoordingsplicht voor de rechtmatigheid van de verwerking](https://minbzk.github.io/Algoritmekader/vereisten/verantwoordingsplicht_rechtmatigheid/index.html)
+- [Verboden toepassingen bij evaluatie of classificatie van personen of groepen personen](https://minbzk.github.io/Algoritmekader/vereisten/verboden_toepassingen_evaluatie_of_classificatie_natuurlijke_personen_of_groepen_personen/index.html)
+- [Verplicht risicobeheersysteem voor hoog-risico AI](https://minbzk.github.io/Algoritmekader/vereisten/verplicht_risicobeheersysteem_voor_hoog_risico_ai/index.html)
+- [Verplichtingen van aanbieders van AI-modellen voor algemene doeleinden](https://minbzk.github.io/Algoritmekader/vereisten/Verplichtingen_van_aanbieders_van_ai_modellen_voor_algemene_doeleinden_/index.html)
+- [Verstrekking van informatie op verzoek](https://minbzk.github.io/Algoritmekader/vereisten/verstrekking_van_informatie_op_verzoek/index.html)
+- [Werknemersvertegenwoordigers en betrokken werknemers worden geïnformeerd door de gebruiksverantwoordelijken die werknemers zijn, voordat een hoog risico AI-systeem wordt ingezet](https://minbzk.github.io/Algoritmekader/vereisten/werknemersvertegenwoordigers_en_betrokken_werknemers_worden_ge%C3%AFnformeerd_inzet_hoog_risico_AI/index.html)
```

Comment:
Dit wil ik echt geautomatiseerd laten gebeuren. Anders is het heel foutgevoelig als we dit op alle pagina's handmatig moeten tonen.  Wat mij betreft wel prima om het beknopter te laten zien ipv de tabel. @rubenrouwhof heeft daar volgens mij ook wel over nagedacht?

Reply:
Ja graag geautomatiseerd!